<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pressure_slider_dialog</class>
 <widget class="QDialog" name="pressure_slider_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pressure Slider</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="parameters_group">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="ScientificDoubleSpinBox" name="k0">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bulk Modulus (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> GPa</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000001000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="read_from_jcpds_button">
        <property name="text">
         <string>Read from JCPDS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="k0_label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Bulk Modulus (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;K&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>k0</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="k0p">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;dK/dP&lt;/span&gt; - the derivative of the bulk modulus with respect to pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000001000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="k0p_label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;dK/dP&lt;/span&gt; - the derivative of the bulk modulus with respect to pressure&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;K'&lt;/span&gt;&lt;span style=&quot; font-style:italic; vertical-align:sub;&quot;&gt;0 &lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>k0p</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="pressure_group">
     <property name="title">
      <string>Pressure</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="pressure">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> GPa</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000100000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="pressure_slider">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure (in GPa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pressure_slider_range_label">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Slider Range:</string>
        </property>
        <property name="buddy">
         <cstring>pressure_slider_range</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScientificDoubleSpinBox" name="pressure_slider_range">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pressure Slider Range&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> GPa</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>k0</tabstop>
  <tabstop>k0p</tabstop>
  <tabstop>read_from_jcpds_button</tabstop>
  <tabstop>pressure_slider_range</tabstop>
  <tabstop>pressure_slider</tabstop>
  <tabstop>pressure</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
